wordpress-mobile/release-toolkit
# Add screenshot tool updates for woo

**To Test:**

This PR should be tested in concert with https://github.com/wordpress-mobile/WordPress-Android/pull/9928. Please view the complete testing steps there!

This PR should be merged once the WPAndroid PR is approved. Next, a `release-toolkit` release should be created. Then we'll update the WPAndroid and merge it.

This PR's tests must pass.
